### PR TITLE
Add display information of field UI

### PR DESCRIPTION
**Pull request by wongpratan**
https://github.com/digi-serve/plugin_ABDesigner/issues/6

## Release Notes

- Add display information of field UI

### CODE REVIEW

File: config/routes.js [hunk 1]
```diff
@@ -173,4 +173,8 @@ module.exports.routes = {
    "get /config/site": "SiteController.configSite",
    "get /config/user": "SiteController.configUser",
    "get /config/inbox": "SiteController.configInbox",
+
+   // System Information
+   // "get /information/object/:objID": "definition_manager/information-object",
+   "get /information/object/:objID/field/:ID": "definition_manager/information-field",
```
Comment (nh758) [inline config/routes.js:179]: Can we make this `/definition/info/... `, so it's consistent with our other `definition_manager` api routes?

**Review by nh758 — APPROVED** [review]
I change then good to go

**Review by guyyoo — APPROVED** [review]
LGTM!

**Review by Hiro-Nakamura — APPROVED** [review]
Good.

I agree with Nathan about keeping the route similar to the other definition_manager ones.

**Pull request merged**